### PR TITLE
Update links to welcome bot images

### LINKED ISSUE #142: Update config.yml file with links to Welcome Bot
The links to the images for the Welcome Bot need to be updated because the images were moved by the owner.

Example of how config.yml file should be updated:  
https://github.com/pymc-devs/pymc/tree/main/.github

Reference: 
https://github.com/pymc-devs/pymc/pull/7107

### PR DESCRIPTION
**Pull request by purna135**
Fixes #142 
Replace the broken welcome bot image link with the new one

### CODE REVIEW

File: .github/config.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Comment to be posted to on first time issues
 newIssueWelcomeComment: >
-  [![Welcome Banner](https://zenodo.org/api/iiif/v2/0c0188d3-d03c-4830-a6e3-00405f5c22fa:8ff47a85-7250-4d86-8e48-2f346b48b2c1:BannerWelcome.jpg/full/750,/0/default.jpg)](https://zenodo.org/record/3695300)
+  ![Welcome Banner](https://raw.githubusercontent.com/pymc-devs/brand/main/welcome-bot/BannerWelcome.jpg)]
```
Comment (reshamas) [inline .github/config.yml:3]:
```suggestion
  ![Welcome Banner](https://raw.githubusercontent.com/pymc-devs/brand/main/welcome-bot/BannerWelcome.jpg)
```

File: .github/config.yml [hunk 2]
```diff
@@ -10,7 +10,7 @@ newIssueWelcomeComment: >
 
 # Comment to be posted to on PRs from first time contributors in your repository
 newPRWelcomeComment: >
-  [![Thank You Banner](https://zenodo.org/api/iiif/v2/0c0188d3-d03c-4830-a6e3-00405f5c22fa:7fbd97cf-283b-480c-b8e1-11866e26245c:BannerThanks.jpg/full/750,/0/default.jpg)](https://zenodo.org/record/3695300)
+  ![Thank You Banner](https://raw.githubusercontent.com/pymc-devs/brand/main/welcome-bot/BannerThanks.jpg)]
```
Comment (reshamas) [inline .github/config.yml:13]:
```suggestion
  ![Thank You Banner](https://raw.githubusercontent.com/pymc-devs/brand/main/welcome-bot/BannerThanks.jpg)
```

File: .github/config.yml [hunk 3]
```diff
@@ -19,7 +19,7 @@ newPRWelcomeComment: >
 
 # Comment to be posted to on pull requests merged by a first time user
 firstPRMergeComment: >
-  [![Congratulations Banner](https://zenodo.org/api/iiif/v2/0c0188d3-d03c-4830-a6e3-00405f5c22fa:32fbdb89-ae1b-434e-830c-88ade86724cc:BannerCongratulations.jpg/full/750,/0/default.jpg)](https://zenodo.org/record/3695300)
+  ![Congratulations Banner](https://raw.githubusercontent.com/pymc-devs/brand/main/welcome-bot/BannerCongratulations.jpg)]
```
Comment (reshamas) [inline .github/config.yml:22]:
```suggestion
  ![Congratulations Banner](https://raw.githubusercontent.com/pymc-devs/brand/main/welcome-bot/BannerCongratulations.jpg)
```

**Comment by reshamas** [comment]
@purna135 Thanks, let's hope it works.

**Comment by purna135** [comment]
Thank you @reshamas

**Comment by purna135** [comment]
image

It still doesn't seem to be working. Do you have any ideas?

**Pull request merged**